server/src/services/db-operations.ts

```diff
@@ -0,0 +1,43 @@
+import { Request, Response } from 'express';
+import { connection } from '../../db';
+
+// return all items for item table
+export const getItems = (req : Request, res : Response) => {
+    console.log('hello')
+    console.log(req.body.id)
+    connection.query('select * from items', (error : any, results : any) => {
+        console.log(results)
+        if (error) {
+            return res.json({ error: error })
+        }
+        return res.json(results);
```

Comment:
this service function responsibility should only focus on business logic, in this case to interact with database and return the array of items back to the routes. the routes then should be the one returning the json results. 

we can have a call to explain why

Reply:
Have fixed using async, await and promises but not sure if the way i did it can be simpler